_maps/map_files/BigRed_v2/BigRed_v2.dmm

```diff
@@ -20614,16 +20604,14 @@
 /area/bigredv2/outside/cargo)
 "bli" = (
 /obj/structure/barricade/wooden{
-	desc = "This barricade is heavily reinforced. Nothing short of blasting it open seems like it'll do the trick, that or melting the breams supporting it...";
-	health = 25000
+	desc = "This barricade is heavily reinforced. Nothing short of blasting it open seems like it'll do the trick, that or melting the breams supporting it..."
 	},
 /turf/open/floor,
 /area/bigredv2/outside/cargo)
 "blj" = (
 /obj/machinery/atmospherics/pipe/simple/green/hidden,
 /obj/structure/barricade/wooden{
-	desc = "This barricade is heavily reinforced. Nothing short of blasting it open seems like it'll do the trick, that or melting the breams supporting it...";
-	health = 25000
+	desc = "This barricade is heavily reinforced. Nothing short of blasting it open seems like it'll do the trick, that or melting the breams supporting it..."
```

Comment:
Unless fish are supporting the barricade it should probably be "beams" instead of "breams" here.